Sources/Rendering/OpenGL/glsl/vtkVolumeFS.glsl

```diff
@@ -906,7 +906,7 @@ vec2 computeRayDistances(vec3 rayDir, vec3 tdims)
 {
   vec2 dists = vec2(100.0*camFar, -1.0);
 
-  vec3 vSize = vSpacing*(tdims - 1.0);
+  vec3 vSize = vSpacing*(tdims);
```

Comment:
you should probably get rid of the parentheses

Reply:
```suggestion
  vec3 vSize = vSpacing*tdims;
```